src/modules/market-price/services/service.ts

```diff
@@ -12,6 +12,13 @@ export class MarketPriceService {
     private coinGeckoService: CoinGeckoService,
   ) {}
 
+  /**
+   * Check if the token is supported by the CoinGecko API
+   */
+  public doesTokenSupportPricingApi(symbol: string): boolean {
+    return !!symbolIdMap[symbol.toLowerCase()];
+  }
+
```

Comment:
nit: could 'isTokenSupportedByPricingApi' be a better name for this func?

Reply:
Yep 👍 Changed it!